### ISSUE TITLE
Short usernames cannot be bridged to discord

### ISSUE THREAD

**Issue by turt2live** [Half-Shot/matrix-appservice-discord#123]
```
ERR! DiscordBot DiscordAPIError: Invalid Form Body
ERR! DiscordBot username: Must be between 2 and 32 in length.
ERR! DiscordBot     at DiscordAPIError (/home/matrix/matrix-appservice-discord/node_modules/discord.js/src/client/rest/DiscordAPIError.js:7:5)
ERR! DiscordBot     at item.request.gen.end (/home/matrix/matrix-appservice-discord/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)
ERR! DiscordBot     at then (/home/matrix/matrix-appservice-discord/node_modules/snekfetch/src/index.js:215:21)
ERR! DiscordBot     at process._tickCallback (internal/process/next_tick.js:109:7)
ERR! DiscordBot  Couldn't send message.  { DiscordAPIError: Invalid Form Body
ERR! DiscordBot username: Must be between 2 and 32 in length.
ERR! DiscordBot     at DiscordAPIError (/home/matrix/matrix-appservice-discord/node_modules/discord.js/src/client/rest/DiscordAPIError.js:7:5)
ERR! DiscordBot     at item.request.gen.end (/home/matrix/matrix-appservice-discord/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)
ERR! DiscordBot     at then (/home/matrix/matrix-appservice-discord/node_modules/snekfetch/src/index.js:215:21)
ERR! DiscordBot     at process._tickCallback (internal/process/next_tick.js:109:7)
ERR! DiscordBot   stack: 'DiscordAPIError: Invalid Form Body\nusername: Must be between 2 and 32 in length.\n    at DiscordAPIError (/home/matrix/matrix-appservice-discord/node_modules/discord.js/src/client/rest/DiscordAPIError.js:7:5)\n    at
item.request.gen.end (/home/matrix/matrix-appservice-discord/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)\n    at then (/home/matrix/matrix-appservice-discord/node_modules/snekfetch/src/index.js:215:21)\n    at process.
_tickCallback (internal/process/next_tick.js:109:7)',
ERR! DiscordBot   name: 'DiscordAPIError',
ERR! DiscordBot   message: 'Invalid Form Body\nusername: Must be between 2 and 32 in length.',
ERR! DiscordBot   path: '/api/v7/webhooks/redacted/redacted?wait=true',
ERR! DiscordBot   code: 50035 }
```

**Comment by Half-Shot** [comment]
While you mention it, it looks like I managed to eat your changes somehow because I can't find them now. I'm going to reapply them :s

**Comment by Half-Shot** [comment]
This was fixed by #124
